commands/inventory/list_agents.go

```diff
@@ -16,9 +16,8 @@
 package inventory
 
 import (
-	"strings"
-
 	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/api/inventorypb"
```

Comment:
If you run `dep ensure -v`, you will notice that a number of dependencies greatly increased. That's because this package pulls the whole gRPC/protobuf stack. We should not use it pmm-admin. Instead, we use only HTTP/JSON APIs.